WowCharComparerWebApp/Controllers/UserControllers/LoginController.cs

```diff
@@ -80,10 +85,12 @@ public IActionResult PasswordRecovery(string userEmail, string userName)
             string recoveryPasswordSubject = "World of Warcraft Character Comparer: Password recovery!";
             string recoveryPasswordBody = $"<p> Hello {userName} </p>" +
                                           $"<p> You've asked to reset password for this World of Warcraft Character Comparer account: {userEmail} </p>" +
+                                          $"<p> Link gonna be validate for {ExpirationTimers.PasswordRecovery} hours</p>" + 
                                           "<p> Please follow link bellow to reset your password: </p>" +
                                           $"<a href=\"{resetPasswordLink}\">Confirmation </a>";
 
             EmailSendStatus emailSendStatus = new EmailManager().SendMail(userEmail, recoveryPasswordSubject, recoveryPasswordBody);
```

Comment:
Add condition: if email has not been sent successful then maybe we should skip adding expiration time?